src/Database/Connection.php

```diff
@@ -299,29 +281,10 @@ public function prepare(Query|string $query): StatementInterface
     public function execute(string $sql, array $params = [], array $types = []): StatementInterface
     {
         return $this->getDisconnectRetry()->run(function () use ($sql, $params, $types) {
-            $statement = $this->prepare($sql);
-            if (!empty($params)) {
-                $statement->bind($params, $types);
-            }
-            $statement->execute();
-
-            return $statement;
+            return $this->_driver->execute($sql, $params, $types);
```

Comment:
This could be reduced to a single line arrow function.

Reply:
Not sure what you mean.

Reply:
```php
$this->getDisconnectRetry()->run(fn () => $this->_driver->execute($sql, $params, $types))
```

Reply:
Ah the new fancy JS type closure which makes vars directly available in it's context.